### DISCUSSION TITLE
flutter web

### DISCUSSION THREAD

**Discussion by aslahmp** [serverpod/serverpod#441] [Q&A] [answered]
how to host flutter web

**Comment by yahu1031** [comment]
@aslahmp If you use firebase, use its hosting and use your own domains to mask the firebase domain. Easy, It shouldn't be complicated as you see(Not related to serverpod though).

**Comment by aslahmp** [comment]
@yahu1031 my domain is linked to server pod aws

**Comment by yahu1031** [comment]
Use flutter build and copy the generated build files to server web folder may be. And host server. (Not so sure)

**Comment by vlidholt** [comment, marked as answer]
The Pixorama example show how you can setup Serverpod's web server to host a Flutter web app. It will also use continuous build and deployment through Github for updating to new versions.

https://github.com/serverpod/pixorama
